hr_expense_receipt_require/readme/CONFIGURE.rst

```diff
@@ -1,3 +1,7 @@
 * Go to Expenses > Configuration > Expense Products
 * Create or select a product
 * Check the box "Require a receipt"
+
+* Go to Expeses > Expense Reports > To Approve
+* Create or select an expense
+* Check the box "Allow lines without attachment"
```

Comment:
```suggestion

Some users may be able to ignore the attachment requirements.
This may be useful where a single document may need to be split into several different expense lines.
The users allowed this must be added to the "Expenses-can skip required attachment" security group.
```